docs/content/security.md

```diff
@@ -379,7 +379,7 @@ allow {                             # Allow request if...
 identity_rights[right] {             # Right is in the identity_rights set if...
     some role
     token := tokens[input.identity]  # Token exists for identity, and...
-    role := token.roles[_]           # Token has a role, and...
+    role = token.roles[_]           # Token has a role, and...
```

Comment:
Please add a whitespace so the comments line up:
```suggestion
    role = token.roles[_]            # Token has a role, and...
```

But also, could you perhaps remove the `some role` line instead? What do you think? I think it'll read more nicely.

Reply:
I think the entire example needs a do-over eventually...so if you only add the space, that's fine.

Reply:
Agreed on removing `some role`. Let's not use unification in docs unless needed.